core/src/main/java/org/apache/accumulo/core/client/admin/servers/ServerType.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.core.client.admin.servers;
+
+public enum ServerType {
+  MANAGER, COMPACTOR, SCAN_SERVER, TABLET_SERVER;
```

Comment:
Note that I did not add `GC` or `MONITOR` here as the client cannot interact with them directly.

Reply:
Why is this one public API? It seems the enum is only used internally for the various Server subclasses. Do we even need this enum at all?